File: CC/Codigos_Fonte_Relatorio_1/micro04/micro04.java

public class micro04{

    public static void main(String[] args){
        int x, num, intervalo = 0;

        

        for(x = 0; x < 5; x++){
            System.out.println("Digite um número: ");
            num = Integer.parseInt(System.console().readLine());
    
            if(num >= 10)
                if(num <= 150) intervalo++;

             

        }
        System.out.println("Ao total, foram digitados "+intervalo+" números no intervalo entre 10 e 150");
    }
}

File: CC/Codigos_Fonte_Relatorio_1/micro04/micro04.pas
program micro04;

var
    x,num,intervalo: integer;


begin
    for x:=1 to 5 do
        begin
            write('Digite um número: ');
            read(num);
    
            if (num >= 10) then
                begin
                if (num <= 150) then intervalo := intervalo + 1;
                end;
        end;

    write('Ao total, foram digitados ',intervalo,' números no intervalo entre 10 e 150');

end.
